lab3/ans/deploy_ans.yaml

```diff
@@ -0,0 +1,303 @@
+version: "3.8"
+
+services:
+  free5gc-upf:
+    container_name: upf
+    image: free5gc/upf:v3.4.2
+    command: bash -c "./upf-iptables.sh && ./upf -c ./config/upfcfg.yaml"
```

Comment:
ip r add default dev `{N6_IF_NAME}` via `{N6_GW_IP}`

Reply:
I write it in **upf-iptable.sh**